var/spack/repos/builtin/packages/r-adegenet/package.py

```diff
@@ -18,13 +18,15 @@ class RAdegenet(RPackage):
     to illustrate various methods."""
 
     homepage = "https://github.com/thibautjombart/adegenet/wiki"
-    url      = "https://cran.r-project.org/src/contrib/adegenet_2.0.1.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/adegenet"
+    url      = "https://cloud.r-project.org/src/contrib/adegenet_2.0.1.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/adegenet"
 
+    version('2.1.1', sha256='3043fe5d731a38ff0e266f090dcda448640c3d0fd61934c76da32d082e5dce7a')
+    version('2.1.0', sha256='7ee44061002b41164bbc09256307ab02e536f4f2ac03f36c7dc8f85f6af4639a')
     version('2.0.1', 'ecb1220ce7c9affaba2987bc7f38adda')
 
-    depends_on('r@2.14:')
```

Comment:
According to https://cran.r-project.org/web/packages/adegenet/index.html, it says: R (≥ 2.14)